cypress/e2e/API/signUp.spec.js

```diff
@@ -0,0 +1,152 @@
+/// <reference types="cypress" />
+
+import { default as homePage } from "../pages/homePage";
+import { default as ModalForms } from "../pages/ModalForms";
+
+describe("SignUp API testing", () => {
```

Comment:
[OPTIONAL] each test section could be called like [DOMAIN]:[FEATURE] or vice versa
example: API : SignUp